### PR TITLE
2 list new space

### PR DESCRIPTION
**Pull request by paulalexrees**
connects to #2

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,7 +2,16 @@ source "https://rubygems.org"
...
-gem "rspec-sinatra"
-gem 'rspec'
 gem 'sinatra'
-gem 'capybara'
+gem 'sinatra-partial'
+gem 'data_mapper'
+gem 'dm-postgres-adapter'
+gem 'rake'
+
+
```
Comment (hanfak) [inline Gemfile]: enter

File: app/app.rb [hunk 2]
```diff
@@ -1,7 +1,17 @@
+ENV['RACK_ENV'] ||= 'development'
+
 require 'sinatra/base'
+# require 'sinatra/partial'
```
Comment (hanfak) [inline app/app.rb]: remove

File: app/app.rb [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
+require 'rubygems'
+require 'data_mapper'
 
-class MakersBnb < Sinatra::Base
+require_relative 'models/data_mapper_setup'
+
```
Comment (hanfak) [inline app/app.rb]: space remove

File: app/views/spaces/index.erb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<ul>
+<% @spaces.each do |space| %>
+
+<li>Name: <%= space.name  %><br>
+  Desc: TBC <br>
+  Price: TBC <br>
+
```
Comment (hanfak) [inline app/views/spaces/index.erb:8]: spaces remove

File: app/views/spaces/index.erb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<ul>
```
Comment (hanfak) [inline app/views/spaces/index.erb]: indent

File: spec/features/view_listings_spec.rb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+# feature 'View listings' do
```
Comment (hanfak) [inline spec/features/view_listings_spec.rb]: ????

**Comment by acookson91** [comment]
:shipit:

**Pull request merged**